### DISCUSSION TITLE
Incorrect type in @types/imapflow

### DISCUSSION THREAD

**Discussion by radams78** [DefinitelyTyped/DefinitelyTyped#57721] [Issues with a @types package]
A type is incorrect in @types/imapflow:

list(): Promise<ListResponse> should be Promise<Array<ListResponse>>

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57834
